packages/gatsby-theme-newrelic/README.md

```diff
@@ -2653,6 +2661,30 @@ With formatting options:
 const formattedCode = useFormattedCode(code, { printWidth: 100 });
 ```
 
+### `useHasMounted`
+
+A hook that determines if a particular component has been loaded in the DOM.
+
+```js
+import { useHasMounted } from '@newrelic/gatsby-theme-newrelic';
+```
+
```

Comment:
Do we need an `Arguments` header here as well? Even if it's `None`

Reply:
added, but to the line **below** 👇